core/genesis.go

```diff
@@ -417,7 +412,6 @@ func DefaultOrchardGenesisBlock() *Genesis {
 func DefaultLocalGenesisBlock() *Genesis {
 	return &Genesis{
 		Config:     params.LocalChainConfig,
-		Knot:       ReadKnot("./core/knot/local_knot.rlp"),
```

Comment:
IMO, this commit should be squashed with the prior commit where you stopped using the knot

Reply:
Squashed it to the previous commit